src/components/Containers/StudentResource/StudentResource.jsx

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import { Text } from "../../Texts";
+import { Button } from "../../Inputs";
+import { media } from "../../../utils";
+
+export const StudentResource = ({
+  titleText,
+  descriptionText,
+  buttons,
+  graphic,
+  graphicLeft,
```

Comment:
since this is a boolean property typically naming conventions starts with ```is``` or ```has``` so I would make this ```isGraphicLeft```

Reply:
and I would also put a default value for this ```isGraphicLeft = true```

Reply:
Makes sense, will change